public/pages/createDetector/components/DataFilters/DataFilter.tsx

```diff
@@ -36,53 +28,46 @@ interface DataFilterProps {
 
 function DataFilter(props: DataFilterProps) {
   return (
-    <ContentPanel
-      title="Data filter"
-      titleSize="s"
-      panelStyles={{ paddingBottom: '0px' }}
-      bodyStyles={{ padding: '0px' }}
-      horizontalRuleClassName={'filters-panel'}
-    >
-      <Fragment>
-        <Field name={`filterType`} validate={required}>
-          {({ field, form }: FieldProps) => (
-            <Fragment>
-              <EuiFlexGroup direction="column" style={{ padding: '10px' }}>
-                <EuiFlexItem style={{ marginBottom: '0' }}>
-                  <EuiFormRow>
-                    <EuiText size="s">
-                      Filters let you choose a subset of data from your data
-                      source. Make a simple filter, or use the Elasticsearch
-                      Query DSL for advanced filters.
-                    </EuiText>
-                  </EuiFormRow>
-                </EuiFlexItem>
-                <EuiFlexItem>
-                  <EuiFormRow
-                    label="Filter type"
-                    isInvalid={isInvalid(field.name, form)}
-                    error={getError(field.name, form)}
-                  >
-                    <EuiSelect
-                      {...field}
-                      options={FILTER_TYPES_OPTIONS}
-                      isInvalid={isInvalid(field.name, form)}
-                    />
-                  </EuiFormRow>
-                </EuiFlexItem>
-              </EuiFlexGroup>
-              <EuiHorizontalRule margin="none" />
-              {field.value === FILTER_TYPES.SIMPLE ? (
-                <SimpleFilter formikProps={props.formikProps} />
-              ) : (
-                <QueryDataFilter />
-              )}
-            </Fragment>
+    <Field name={`filterType`} validate={required}>
+      {({ field, form }: FieldProps) => (
+        <Fragment>
+          <EuiFormRow
+            fullWidth
+            label={
+              <div>
+                <p>
+                  Data fiter
+                  <span className="optional">- optional</span>
+                </p>
+                <p className="sublabel">
+                  Choose a subset of your data source to focus your data stream
+                  and reduce noisy data.
+                </p>
+                <p className="sublabel">
+                  Use the visual editor to create a simple filter, or use the
+                  Elasticsearch query DSL to create more advanced filters.
+                </p>
+              </div>
+            }
```

Comment:
Recommend to use Text from EUI to be consistent with sizing and their formattings.

Reply:
I tried to use EUIText, but it is not easy to make it work with EuiFormRow, so I did these formatting inside label.